lib/cc/cli/init.rb

```diff
@@ -26,13 +26,19 @@ def create_codeclimate_yaml
           config["ratings"]["paths"] |= engine_config["default_ratings_paths"]
         end
 
-        if filesystem.exist?("vendor")
-          config["exclude_paths"] = ["vendor/**/*"]
-        end
+        config["exclude_paths"] = autoexclude_paths
```

Comment:
I think it might read a little cleaner if you call the method `exclude_paths`  and have it take in an argument `AUTO_EXCLUDE_PATHS`